src/migration-wizard/steps/discovery/assessment-report/NetworkTopology.tsx

```
@@ -13,29 +11,22 @@ interface NetworkTopologyProps {
-            padding: 2,
-          }}
-        >
+      <CardBody style={{padding: 0}}>
+      <div style={{ maxHeight: tableHeight, overflowY: 'auto', overflowX:'auto',padding: 2 }}>
           <ReportTable<InfraNetworksInner>
             data={networks}
             columns={['Type', 'VlanId']}
             fields={['type', 'vlanId']}
-            withoutBorder
-          />
+            withoutBorder          
```

ESLint

[error] Delete `··········`

(prettier/prettier)